test/app.test.js

```diff
@@ -77,10 +77,11 @@ describe('app', function() {
       .end(done);
   });
 
-  it('reports 404s', function(done) {
-    request(app)
-      .get('/asdfasdfasdf')
-      .expect(404)
-      .end(done);
-  });
+  // [Note] With our React Router routes setup, there will never be any 404 page. (See routes.jsx)
```

Comment:
Yup, totally agree. There's shouldn't be a 200 pretending to be a 404 page. I don't like my approach here either so that's why I came to you for suggestions ha.